fixes.inc

```diff
@@ -5062,6 +5100,16 @@ static stock
 	 */
 #if FIX_PassengerSeating
 	FIXES_gsPSTimer[MAX_PLAYERS],
+#endif
+	/**
+	 * <remarks>
+	 * The vehicle's stereo used in GetVehicleComponent.
+	 * </remarks>
+	 */
+#if FIX_GetVehicleComponentInSlot
+	FIXES_gsStereo[MAX_VEHICLES],
+	FIXES_gsBumpers[MAX_VEHICLES][2],
+	FIXES_gsBullBars[MAX_VEHICLES][2];
```

Comment:
In normal circumstances, use five separate variables, there's no need for an array for the bumpers/bull bars.  However, there is absolutely no need for 5 variables per vehicle - you're only storing five true/false values, either they have the component or they don't.  That can be done in a single variable.

Reply:
No. I'm storing model of components. Used later, to return them.